multisig-wallet/.docs/sequence-diagram.drawio

```diff
@@ -0,0 +1 @@
+<mxfile host="app.diagrams.net" modified="2023-01-30T14:24:08.529Z" agent="5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36" etag="F4oPFbVkD1hfKviJhd6I" version="20.8.13" type="device"><diagram name="Page-1" id="5R1fmDkTnkg9pyA9cscx">7Vttj5s4EP41kXofrgKcAPnYzW5fdD2p0vbUly8rLzhA6+DImCbpr+8YTAiYzXFKct5Av2zw2Bgzz8x4nsE7QYvV9g3H6/hvFhI6caxwO0G3E8exLWsGP1KyqyRzv5REPAmVrBbcJz9JNVBJ8yQkWWOgYIyKZN0UBixNSSAaMsw52zSHLRltPnWNI6IJ7gNMdemnJBRxKfUdr5a/JUkUV0+23XnZs8LVYPUmWYxDtjkQobsJWnDGRHm12i4Ildqr9FLe9/qJ3v3COElFnxveL1/lf0XvnC/TTHz8Nku+RvTDn2qWH5jm6oXVYsWu0gBneRoSOYk1QTebOBHkfo0D2bsB0EEWixWFlg2XajrCBdk+uU57//ZgN4StiOA7GLJtIq8sBqkJNrX2bctTY+ID1c+REmIFebSfutYKXCjF/AcleZpOSAhGopqMi5hFLMX0rpbeNLVWj3nP2Frp6hsRYqcsHueCNTUJ2uK7z+r+ovFFNl7Oqubt9rDzdqda5VrlAo/rH96H5TwgR95b2YLAPCLiyDi3G09OKBbJj+Y6zg7O/Dc4R8HxTYLjaOEF4nQmeB4Ixl/8cVqsWSaULhhlvLgXhZj4ywDkMD/7Tg563MAnj8szRSe/GZ5mbkd8cjrCk3up6DQdqQOgng7gmHQA1LG/uhSWe/MIF5G8+CcjvBLCM/ZyDVYwWtFEoWnpKUtJyy2UCNMkSqEZgJLhYehGukACSc4r1bFKwrAwjC6XaxrLGXxo1nIhz9VcyO3woIvt77NTQHLxSiopfczkT0ZEBlPl6zIlFRxDTjpYILVg6PQMhheD0p6NNBq6PaOhjUyGQ9v9Dc9xeIzuVvZYmU5veKYm4XG1fQq2hYBcZR49tZ5bHu1r2g0waJc+wCaeZrCPJywdhKr9KTKs6qrEc46MC0CSuVYBlXThmMDfVGZY0KiRG24SpsFr9YT3cklYV1XxJHhlVK6wDXJeqAZQltFvPMCaz66dsdYa7L7Vtqcg/Z/KbWNlP/3xMZtf6/XQgy3qIcZZfJUZhtuq6CDPdDJXhcDxeULvsqhn1BP0wqiIOcliRsPh+IF5UuOM9fOYPe3rB0Y/kFXLPPCDTWEXw3ECb2raCWyd2i8wDXIAWBIKqemJPBrCpfFtIfUHWy84JaWZBsBg6IW2aVdnWszxxnkfnCDYADYxTiNScEK1a4wHqGfAA/X6zb8ClReE3yqj24jcyrOMo6WXYwrWsSTXeQTD91qBq2/B63JZls7ryJYEuSBXXzxuK3taZZTmlH3SmYqu6qLCSi52oorGS3n/UENUG1OnL025XIjSP22dH9OhfwjQ4qLvm4bV02DNiHjY52yDiIjPgOTr/EaqWaVag1CyeRKJ9Czq1G+WNYcZCZPRXMc4k0F65nZeVAfPdzRPNc53kJ4g4vWaM2irmHiVIbF92NK2HNMhUS9evik+1Jdahgu2VAlYyf6H6gNtaFzTlTSkH2l+xBRf62Gxtn5hdtOmrxOW0vSVmve2nxZPz8iQT4F7fjMwzU3zDqTzjuvmHG0HQK7pKgzSOUfpACXxLoj2GFLatvEj1FWMdM5i/NCs/1246Dv4r2t09ws=</diagram></mxfile>
```

Comment:
Was this file supposed to be added?

Reply:
yh, we can load it up online and update the diagram without having to remake it from scratch each time